### PR TITLE
Freetypes ui

### PR DESCRIPTION
**Pull request by darconny**
UI work:
- flexible scorecards
- added arrow to freetypes selector button
- advertisement local fixes
- Blog list Firefox fix (truncating title)

### CODE REVIEW

**Comment by nistormihai** [comment]
LGTM

File: client/app/scripts/liveblog-edit/freetype.service.js [hunk 1]
```diff
@@ -475,7 +475,7 @@
...
...al"><form ng-submit="ft.upload(ft.preview)" name="uploadImageForm"><div class="modal-header"><button class="close" ng-click="ft.closeUploadModal()"><i class="icon-close-small"></i></button> <h3 translate>Upload Advertisment Image</h3> </div><div class="modal-body"> <div class="upload-form"> <section class="main" sd-image-preview="ft.preview.url" data-file="ft.preview.img" data-progress-width="ft.progress.width" > <div class="upload-progress" ng-show="progress.width"> <div class="bar" style="width:{{progress.width}}%;"></div></div><div class="computer" ng-if="!ft.preview.url" ng-hide="ft.progress.width"> <div class="dropzone" ng-file-drop="ft.preview.img=$files[0]"> <div class="text" translate>Drop it here</div><div class="input-holder"> <input type="file" accept="image/*;capture=camera" ng-file-select="ft.preview.img=$files[0]"> </div></div></div><div class="preview" ng-if="ft.preview.url"> <div class="original"> <div sd-plain-image data-src="ft.preview.url" data-progress-width="ft.progress.width" data-file="ft.preview.img" ></div></div></div></div></div><div class="modal-footer"> <button type="submit" class="btn btn-primary btn-info" translate>Upload</button> </div></form></div>',
```
Comment (m0g) [inline client/app/scripts/liveblog-edit/freetype.service.js]: Isn't there a way to make that template more readable?

Comment (jey07ro) [inline client/app/scripts/liveblog-edit/freetype.service.js]: @vladnicoara is takeing care of this, but not in this PR.

Comment (nistormihai) [inline client/app/scripts/liveblog-edit/freetype.service.js]: @vladnicoara is taking care of this, but not in this PR.

**Pull request merged**